.changeset/popular-roses-deliver.md

```diff
@@ -0,0 +1,41 @@
+---
+'@makeswift/runtime': patch
+---
+
+Add RichText control.
+
+Until now, having rich text within your custom components required putting a Text component within a Slot control.
+While we are greatful for the flexibility of our primitives we wanted to enable specific component API design.
```

Comment:
typo: "greatful" -> "grateful"

On a separate note, it seems weird for us to be grateful for something we built ourselves for other people haha. Maybe we can say something like "While it's possible to add rich text by dropping a `Text` component in a slot, sometimes you've got a component where text is the only thing you want to be editable, not any arbitrary React node"

Reply:
I like that